plugins/qiskit_executor/routes.py

```diff
@@ -354,39 +358,7 @@ def post(self, arguments, db_id: int):
         db_task.clear_previous_step()
         db_task.save(commit=True)
 
-        if not params.backend:
-            # start the backend selection task
-            step_id = "backend-selection"
-            href = url_for(
-                f"{QISKIT_EXECUTOR_BLP.name}.BackendSelectionView",
-                db_id=db_task.id,
-                _external=True,
-            )
-            ui_href = url_for(
-                f"{QISKIT_EXECUTOR_BLP.name}.BackendSelectionFrontend",
-                db_id=db_task.id,
-                _external=True,
-            )
-            task = add_step.s(
-                db_id=db_task.id,
-                step_id=step_id,
-                href=href,
-                ui_href=ui_href,
-                prog_value=2,
-                prog_target=3,
-                prog_unit="steps",
-                task_log="Starting backend selection step",
-            )
-        else:
-            # start the execution task
-            task: chain = (
-                start_execution.s(db_id=db_task.id)
-                | result_watcher.si(db_id=db_task.id)
-                | save_task_result.s(db_id=db_task.id)
-            )
-        # save errors to db
-        task.link_error(save_task_error.s(db_id=db_task.id))
-        task.apply_async()
+        start_execution.s(db_id=db_task.id).apply_async()
```

Comment:
It is probably a good idea to handle the case where the token is loaded from the env variable (see line 314) before this line in the `post` function so that the task will always see a correct token (or no token) and not `****`